tests/online_migration/sle12_online_migration/orphaned_packages_check.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check and output orphaned packages
+# Maintainer: Wes <whdu@suse.com>
+# Tags: poo#19606
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+
+
+sub run() {
+    select_console 'root-console';
+
+    #No orphaned packages list to compare currently, so we simply output the result
+    script_run("zypper packages --orphaned |tee /dev/$serialdev");
```

Comment:
Better to use `zypper_call('packages --orphaned', log => 'orphaned.log');` I think :)